diff_result.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
+class DiffResult:
+    def __init__(self):
+        self.files = []
+        self.regions = 0
+        self.lineAdded = 0
+        self.lineDeleted = 0
+        self.functionCalls = defaultdict(int)
+
+    def toText(self):
```

Comment:
I'm not a hardcore pythonista, but I thought `camelCase` was strongly discouraged in favour of `snake_case`, if you were rewriting the project why not stick to the standard?

Reply:
You're right, this is a habit that I should be easy to kick out with the help of linters